content/documentation/getting-started/index.adoc

```diff
@@ -367,6 +367,17 @@ namespaces:
     - mycorp_test.*
 ----
 
+==== Namespace Selectors
+
+Kiali supports an optional label selector for namespaces which is used to fetch a subset of the available namespaces. The label selector is defined under the namespaces definition. The example below selects all namespaces that have a label `kiali-enabled: true`.
```

Comment:
( While not the general style of the file, it is a good idea to start a new sentence on a new line  -- no need to change it now )

Would it make sense to show the command to label a namespace with that selector for people that are less familiar? Or link to the kube documentation?

Reply:
I added it